include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -5938,7 +5938,7 @@
       "version" : "None"
     },
     {
-      "opname" : "OpReportIntersectionKHR",
+      "opname" : "OpReportIntersectionNV",
```

Comment:
Not for this change but this makes me wonder whether we shouldn't introduce a way of encoding aliases that does not require duplicating whole definitions. The first idea that comes to mind is something like:

```
{
  "opname": "OpReportIntersectionKHR",
  "aliases": ["OpReportIntersectionNV"]
}
```

Is there a reason we wouldn't want to do something like this?

Reply:
That's the topic of #138

Reply:
My opinion is that we should do something along those lines, but it will take longer to nail down how we want it to work (e.g. in conjunction with the spec). So for now this is desirable.

Reply:
Yeah, it's a much bigger change. I'm not suggesting we try to do this here and now, just checking that I hadn't missed a reason we couldn't (or didn't want to) go there.